aggregator/src/main/resources/application.properties

```diff
@@ -37,6 +37,10 @@ quarkus.log.sentry.dsn=FILL_ME
 
 %test.quarkus.scheduler.enabled=false
 
+# if you change this value, you should also change:
+# notifications.backend.email.subscription.periodic.cron.enabled in notifications-backend project
+# to avoid both cron jobs are running at the same time
+notifications.aggregator.email.subscription.periodic.cron.enabled=false
 notifications.aggregator.email.subscription.periodic.cron=0 0 0 * * ?
```

Comment:
The scheduled job starts at 2 am in `backend` while it starts at 12 am here. Is there any reason for that change?

Reply:
The property: notifications.backend.email.subscription.daily.cron=0 0 2 * * ? is the same as the whole time. I just renamed it.

The new property is also the same as before. I mean Thomas suggested that time. But anyway, we will change it anyway as soon as it's on ci to something like "every 1-2 minutes" or so to be able to test it.

I would say the current time in the new cron job is not that important for now as it will be changed at least two times anyway.

Reply:
I'm only concerned about prod:

- [Current prod cron value](https://github.com/RedHatInsights/notifications-backend/blob/56c53f81c8f24f0cf35f124298ab816879c359a5/backend/src/main/resources/application.properties#L93)  is `0 0 2 * * ?` (2 am)
- [Backend cron value after merge](https://github.com/bennetelli/notifications-backend/blob/22fc8a207ed4fe1b8a6d9bc62d528dc51ac44e70/backend/src/main/resources/application.properties#L102) will be `0 0 2 * * ?` (same as current prod)
- [Aggregator cron value after merge](https://github.com/bennetelli/notifications-backend/blob/22fc8a207ed4fe1b8a6d9bc62d528dc51ac44e70/aggregator/src/main/resources/application.properties#L44) will be `0 0 0 * * ?` (two hours earlier than current prod)

I would expect these to be all equal unless there's a good reason to change the execution hour.

Reply:
I didn't check if this is overridden using env vars on prod though.

Reply:
But aggregator won't get deployed to prod after merged this PR. There will be a second PR after we got it up and running on ci as I am sure there will be some things to clean up, maybe some things won't work, ... and so on. 
And we will also have to write the aggregator in reactive so we can get the benefits of the models project.

Reply:
Ok let's take care of that later then.